docker/auto-setup.sh

```diff
@@ -6,6 +6,7 @@ set -eux -o pipefail
 
 DB="${DB:-cassandra}"
 SKIP_SCHEMA_SETUP="${SKIP_SCHEMA_SETUP:-false}"
+SKIP_DB_CREATE="${SKIP_DB_CREATE:-false}"
```

Comment:
you can write these like:
```
: ${DB:=cassandra}
: ${SKIP_SCHEMA_SETUP:=false}
: ${SKIP_DB_CREATE:=false}
```
to avoid a little repetition

Reply:
I will address this in separate PR.